docs/install-network-operator-gov-ready.rst

```diff
@@ -0,0 +1,211 @@
+The NFD container is built on top of a scratch image, providing a highly secure container environment.
+For information on NFD CVEs and security updates, refer to the `NFD GitHub repository <https://github.com/kubernetes-sigs/node-feature-discovery/security>`_.
+
+Refer to the NFD documentation for `installation instructions <https://kubernetes-sigs.github.io/node-feature-discovery/stable/get-started/index.html>`_.
+
+
+.. _create-ngc-api-pull-secret:
+
+Create NGC API Pull Secret
+---------------------------
+
+Add a Docker registry secret for downloading the Network Operator artifacts from NVIDIA NGC in the same namespace where you are planning to deploy the NVIDIA Network Operator.
+Update ``ngc-api-key`` in the command below with your NGC API key.
+
+.. code-block:: console
+
+   $ kubectl create secret -n network-operator docker-registry ngc-secret \
+       --docker-server=nvcr.io \
+       --docker-username='$oauthtoken' \
+       --docker-password=<ngc-api-key>
+
+.. _create-ubuntu-pro-token-secret:
+
+Create Ubuntu Pro Token Secret
+-------------------------------
+
+Create a Kubernetes secret to hold the value of your Ubuntu Pro token secret. 
+This secret will be used in the install command in the next step.
+
+The Ubuntu Pro Token is required for the driver container to download kernel headers and other necessary packages from the Canonical repository when using the FIPS-enabled kernel on Ubuntu 24.04.
+
+.. _deploy-nvidia-network-operator-gov-ready:
+
+Install NVIDIA Network Operator Government-Ready Components
+--------------------------------------------------------
+
+#. Label your ``network-operator`` namespace for the Operator to set the enforcement policy to privilege.
+
+   .. code-block:: console
+
+      $ kubectl label --overwrite ns network-operator pod-security.kubernetes.io/enforce=privileged
+
+#. Add the NVIDIA Helm repository:
+
+   .. code-block:: console
+
+      $ helm repo add nvidia https://helm.ngc.nvidia.com/nvidia \
+          && helm repo update
+
+#. Install the NVIDIA Network Operator with SR-IOV Network Operator.
+
+   .. code-block:: console
+
+      $  helm install network-operator nvidia/network-operator \
+           --namespace network-operator \
+           --set sriov-network-operator.images.sriovConfigDaemon=doca-driver-stig-fips \
+           --set sriov-network-operator.imagePullSecrets={ngc-secret} \
+           --set sriovNetworkOperator.enabled=true
+           --set nfd.enabled=true
+
+.. _update-ubuntu-pro-token-in-nicclusterpolicy:
+
+Update Ubuntu Pro Token in NicClusterPolicy
+=========================================
+
+You can update your Ubuntu Pro Token after installation by editing your Ubuntu Pro Token secret.
+This secret name is set as value of ``ofedDriver.env`` and ``UBUNTU_PRO_TOKEN`` environment variable of the Network Operator NicClusterPolicy.
```

Comment:
From https://github.com/Mellanox/doca-driver-build/pull/93 , the var is used as is. No one is pulling it out of the secret